src/components/lib/FlowGrid/FlowGrid.jsx

```diff
@@ -129,16 +131,21 @@ export default class FlowGrid extends Component{
     return Math.max(6, lowestItem, selected) || 6;
   }
 
+  _isWithin(location) {
```

Comment:
We have this function (or an equivalent) in many places; in a separate PR I'm making a location utils file with it and a few others, so eventually I'll strip this function out. Just FYI.